requirements/requirements-dev.in

```diff
@@ -22,6 +22,13 @@ pylint
 mypy
 black==21.6b0
 typed-ast>=1.4.2
+types-requests
```

Comment:
Could we also just use the `mypy --install-types` feature or is it better to explicitly have the stub packages as requirement?

Reply:
Yes, I was thinking about that. Not sure what would be best -- this way we have all dependencies explicitly listed, but on the other hand the list may only grow in the future.

Reply:
We decided to just leave it as is for now.